Documentation/contributing.rst

```diff
@@ -565,6 +582,27 @@ Best Practices for Writing Tests
         vm.ReportFailed()
     }
 
+
+Ginkgo Extensions
+~~~~~~~~~~~~~~~~~~
+
+In cilium, some Ginkgo features are extended to cover some uses cases that are
+useful for how cilium run test
+
+BeforeAll
+^^^^^^^^^^
+
+This function will run before all BeforeEach within a describe or context. This
+function is the new ``SetUp`` or initalize function for our Ginkgo test.
+
+AfterAll
+^^^^^^^^^
+
+This function will run after all ``AfterEeach`` functions defined in a Describe or
+Context. This function is the ``tearDown`` or the ``finilize`` function. The best
```

Comment:
Using \`\`\ makes it seem like the contents between the ticks are official terms, but they aren't. Remove ``tearDown`` and ``finilize`` and say something like:

This is used for tearing down objects created which are used by all `It`s within the given `Context` or `Describe`.  It is ran after all `It`s have ran.